servicecontrol/license.md

```diff
@@ -19,6 +19,10 @@ The app will import the designated license file into the registry. The license f
 ![](managementutil-addlicense.png 'width=500')
 
 
+#### Imported locations
+ 
+The license is then imported at *the machine level* in the registry (`HKLM\SOFTWARE\ParticularSoftware\@License`) and on the file system (`%ProgramData%\ParticularSoftware\license.xml`).
```

Comment:
Why the special formatting on "the machine level" ?

Reply:
@bording machine and user level emphasized because it is either user or machine level. This is what the sentence is actually about. The rest of the sentence basically only exists to document it in a readable way.